### PR TITLE
Update Media Player overview

### PR DESCRIPTION
**Pull request by jamesdonoh**
Progresses https://github.com/bbc/simorgh-infrastructure/issues/750

**Overall change:** 
_Updates the media player overview to add more information about the reasons for the approach and remove hostname-specific references_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [NA] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [NA] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [no] This PR requires manual testing

### CODE REVIEW

File: docs/Media-Player-Guide.md [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
+Simorgh is designed to be AMP-first and all features are built to work for both AMP and canonical page users. At the time of writing the only viable method of including the BBC's Standard Media Player (SMP) on an AMP page is using the [amp-iframe](https://amp.dev/documentation/components/amp-iframe/) element and a custom _embed endpoint_ to serve the player itself.
 
-_*Note that it is possible to use this implementation without a placeholder, just don't pass anything to the `placeholder` prop, and the media player will load in a frame on page load._
+To minimise the duplication that would arise from having two very different media player implementations it was decided use an iframe-based approach for both AMP and canonical pages. The responsibility for configuring and rendering the media player was therefore moved largely outside of Simorgh into `news-av-player`. This simplifies the code and helps to reduce the testing and maintenance burden of supporting media on both platforms.
```
Comment (HarveyPeachey) [inline docs/Media-Player-Guide.md]:
```suggestion
To minimise the duplication that would arise from having two very different media player implementations it was decided to use an iframe-based approach for both AMP and canonical pages. The responsibility for configuring and rendering the media player was therefore moved largely outside of Simorgh into `news-av-player`. This simplifies the code and helps to reduce the testing and maintenance burden of supporting media on both platforms.
```

**Review by HarveyPeachey — APPROVED** [review]
LGTM after small typo is amended. Has anything been added to the runbook?

**Comment by jamesdonoh** [comment]
Run Book is next on my list 😛

**Review by amywalkerdev — APPROVED** [review]
Looks good!

**Pull request merged**